io.crow_translate.CrowTranslate.yaml

```diff
@@ -0,0 +1,91 @@
+id: org.kde.CrowTranslate
+sdk: org.kde.Sdk
+runtime: org.kde.Platform
+runtime-version: 5.15-23.08
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --socket=pulseaudio
+  - --share=network
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.kde.StatusNotifierWatcher
+  # The app supports capturing screen via KWin and Mutter DBus APIs, but
+  # access to it seems overly broad in spite of the app also supporting XDP API
+  #- --talk-name=org.kde.KWin
+  #- --talk-name=org.gnome.Shell
+rename-icon: crow-translate
+copy-icon: true
+command: crow
+cleanup:
+  - '*.a'
+  - '*.la'
+  - /include
+  - /lib/cmake
+  - /lib/pkgconfig
+modules:
+  - name: crow-translate
+    buildsystem: cmake-ninja
+    sources:
+      - type: git
+        url: https://github.com/crow-translate/crow-translate.git
+        tag: 2.11.1
+        commit: 7113ff01a994584ed0a3ec557fc3a694fcb06b63
+        x-checker-data:
+          type: git
```

Comment:
This repo is archived, I'm guessing this should be using KDE provided tarballs.

Reply:
Also is this the source now https://invent.kde.org/office/crow-translate ? (That'd justify the org.kde. ID)

Reply:
> This repo is archived, I'm guessing this should be using KDE provided tarballs.

It's because we don't have a new release yet. But I want to create a flathub entry first, otherwise [the app page](https://apps.kde.org/crowtranslate/) will point to an old release.

> 

> Also is this the source now https://invent.kde.org/office/crow-translate ? (That'd justify the org.kde. ID)

Yes!

Reply:
Then please use invent as the source, with a fixed commit, not GitHub

Reply:
Switched to using invent. I realized that we imported previous releases, so I can just change the URL and it will work.